app/javascript/hooks/queries/admin/roles/useRoles.jsx

```diff
@@ -2,7 +2,7 @@ import { useQuery } from 'react-query';
 import { useSearchParams } from 'react-router-dom';
 import axios from '../../../../helpers/Axios';
 
-export default function useRoles(search, enabled = true) {
+export default function useRoles({ search = '', enabled = true } = {}) {
```

Comment:
This was felt better to have for verbosity.
So, instead of having `useRoles('', canUpdateRole)` which is less informational and requires being aware of the `useRoles` function signature having `useRoles({ enabled: canUpdateRole })` is more verbose and informational.